Frontend/src/pages/dashboard/components/Retailer/recent-orders.jsx

```
@@ -1,47 +1,102 @@
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    setNewOrder((prev) => ({ ...prev, [name]: value }));
+  };
+
+  // Add new order
+  const handleAddOrder = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post("http://localhost:3000/api/retailer-orders", {
+        ...newOrder,
+        price: `$${parseFloat(newOrder.price).toFixed(2)}`, // Ensure price format
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)